nixos/doc/manual/release-notes/rl-1909.xml

```diff
@@ -131,6 +131,15 @@
       and <option>services.limesurvey.virtualHost</option> options.
     </para>
    </listitem>
+   <listitem>
+    <para>
+      The setopt declarations will be evaluated at the end of zshrc, so any code in interactiveShellInit,
+      loginShellInit and promptInit may break if it relies on those options being set.
+    </para>
+    <para>
+      All options can be referenced using <xref linkend="opt-programs.zsh.setOptions" />.
```

Comment:
This doesn't need to be a part of the changelog and can be removed. I meant with the sentence that the documentation for `loginShellInit`, `interactiveShellInit` etc can be referenced here using the `xref` tag :)